modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts

```diff
@@ -19,17 +20,26 @@ import { renderWordCount } from './WordCount';
 const renderStatusbar = (editor: Editor, providersBackstage: UiFactoryBackstageProviders): SimpleSpec => {
 
   const renderBranding = (): SimpleSpec => {
-    const label = I18n.translate([ 'Powered by {0}', 'Tiny' ]);
-    const logo = `<svg width="50" height="16" viewBox="0 0 50 16" xmlns="http://www.w3.org/2000/svg" title="${label}">
-      <path fill-rule="evenodd" clip-rule="evenodd" d="M10.143 0c2.608.015 5.186 2.178 5.186 5.331 0 0 .077 3.812-.084 4.87-.361 2.41-2.164 4.074-4.65 4.496-1.453.284-2.523.49-3.212.623-.373.071-.634.122-.785.152-.184.038-.997.145-1.35.145-2.732 0-5.21-2.04-5.248-5.33 0 0 0-3.514.03-4.442.093-2.4 1.758-4.342 4.926-4.963 0 0 3.875-.752 4.036-.782.368-.07.775-.1 1.15-.1Zm1.826 2.8L5.83 3.989v2.393l-2.455.475v5.968l6.137-1.189V9.243l2.456-.476V2.8ZM5.83 6.382l3.682-.713v3.574l-3.682.713V6.382Zm27.173-1.64-.084-1.066h-2.226v9.132h2.456V7.743c-.008-1.151.998-2.064 2.149-2.072 1.15-.008 1.987.92 1.995 2.072v5.065h2.455V7.359c-.015-2.18-1.657-3.929-3.837-3.913a3.993 3.993 0 0 0-2.908 1.296Zm-6.3-4.266L29.16 0v2.387l-2.456.475V.476Zm0 3.2v9.132h2.456V3.676h-2.456Zm18.179 11.787L49.11 3.676H46.58l-1.612 4.527-.46 1.382-.384-1.382-1.611-4.527H39.98l3.3 9.132L42.15 16l2.732-.537ZM22.867 9.738c0 .752.568 1.075.921 1.075.353 0 .668-.047.998-.154l.537 1.765c-.23.154-.92.537-2.225.537-1.305 0-2.655-.997-2.686-2.686a136.877 136.877 0 0 1 0-4.374H18.8V3.676h1.612v-1.98l2.455-.476v2.456h2.302V5.9h-2.302v3.837Z"/>
-    </svg>`;
-    const linkHtml = `<a href="https://www.tiny.cloud/?utm_campaign=editor_referral&amp;utm_medium=poweredby&amp;utm_source=tinymce&amp;utm_content=v5" rel="noopener" target="_blank" tabindex="-1" aria-label="${label}">${logo}</a>`;
     return {
       dom: {
         tag: 'span',
         classes: [ 'tox-statusbar__branding' ],
-        innerHtml: linkHtml
-      }
+      },
+      components: [
+        {
+          dom: {
+            tag: 'a',
+            attributes: {
+              'href': 'https://www.tiny.cloud/powered-by-tiny?utm_campaign=editor_referral&utm_medium=poweredby&utm_source=tinymce&utm_content=v6',
+              'rel': 'noopener',
+              'target': '_blank',
+              'tabindex': '-1',
```

Comment:
Is there a particular reason for making this link unfocusable via keyboard? I might be wrong but it seems that combining both `tabindex=-1` and `aria-label` attributes is meaningless since screen reader won’t be able to read the lable without the link getting focused in the first place

Reply:
Oh, and if we decide to remove `tabindex` attribute it would be a good idea to add `role="presentation"` to the svg as @spocke suggested previously.

Reply:
The tabindex=-1 is required because it's handled via our UI framework (Alloy) not via native keyboard navigation. That said though, now this is an Alloy spec this isn't quite the correct way to do this. Instead, we should be adding the `Focusing` behaviour instead.

Reply:
I didn't add or remove any attributes, I just converted the existing attributes to Alloy spec:
https://github.com/tinymce/tinymce/blob/8c13482153500dba4a1b16c033c190cc72b91417/modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts#L23
This discussion makes me happy, as does the new commit to change the URL. These problems existed before but were not noticed.

Reply:
Sorry I had noticed that as well and should have noted they were there beforehand in my reply as well 😞

Reply:
Was talking to @TheSpyder about this via slack. I've gone ahead and swapped it to use the `Focusing` behaviour as I mentioned.

As for the `role="presentation"` part, if that was an a11y issue then it'd also be an issue for all our toolbar icons, etc... as well so we should probably investigate that and follow up in a separate Jira instead. Does anyone have any objections to following that up via a separate Jira?

Now lastly, and this isn't related to these changes but more the original change, but the is no focus style for the logo so you can't see what's focused. That does seem like an a11y issue so we probably need to chase that up with the design team and see if some styles should be added for that. We should handle that after this PR is merged into the existing PR though.